scene/animation/animation_player.h

```diff
@@ -71,7 +71,7 @@ class AnimationPlayer : public Node {
 private:
 	enum {
 
-		NODE_CACHE_UPDATE_MAX = 1024,
+		NODE_CACHE_UPDATE_MAX = 2048,
```

Comment:
Might also be better in its own PR with explanation for why it's needed.

Reply:
Maybe this should an editor setting.

I've had to bump it arbitrarily when the FBX is too complicated.

Reply:
Removed.